test/lib/usd/importExport/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+set(TARGET_NAME IMPORT_EXPORT_TEST)
+
+set(test_script_files
+    testUsdExportAsClip.py
+    testUsdExportCamera.py
+    testUsdExportColorSets.py
+    testUsdExportConnected.py
+    testUsdExportDisplayColor.py
+    testUsdExportEulerFilter.py
+    testUsdExportFilterTypes.py
+    testUsdExportFrameOffset.py
+    testUsdExportInstances.py
+    testUsdExportLocator.py
+    testUsdExportMesh.py
+    testUsdExportNurbsCurve.py
+    testUsdExportOpenLayer.py
+    testUsdExportOverImport.py
+    testUsdExportParentScope.py
+    # To investigate: following test asserts in MFnParticleSystem, but passes.
+    # PPT, 17-Jun-20.
+    testUsdExportParticles.py
+    testUsdExportPref.py
+    # Following test runs only in legacy render layer mode, otherwise skipped.
+    # Legacy render layers are deprecated, and should not be used.
+    # To avoid skipping and run in legacy render layer mode,
+    # export MAYA_ENABLE_LEGACY_RENDER_LAYERS=1
+    testUsdExportRenderLayerMode.py
+    ## XXX: This test is disabled by default since it requires the RenderMan for Maya plugin.
+    # testUsdExportRfMLight.py
+    testUsdExportSelection.py
+    testUsdExportShadingInstanced.py
+    testUsdExportShadingModeDisplayColor.py
+    testUsdExportShadingModePxrRis.py
+    testUsdExportSkeleton.py
+    testUsdExportStripNamespaces.py
+    testUsdExportVisibilityDefault.py
+    testUsdImportCamera.py
+    testUsdImportColorSets.py
+    # To investigate: following test asserts in TDNshapeEditorManager.cpp, but
+    # passes.  PPT, 17-Jun-20.
+    testUsdImportFrameRange.py
+    testUsdImportMayaReference.py
+    testUsdImportMesh.py
+    # XXX: This test is disabled by default since it requires the RenderMan for Maya plugin.
+    # testUsdImportRfMLight.py
+    testUsdImportSessionLayer.py
+    testUsdImportShadingModeDisplayColor.py
+    testUsdImportShadingModePxrRis.py
+    testUsdImportSkeleton.py
+    testUsdImportXforms.py
+)
+
+add_custom_target(${TARGET_NAME} ALL)
+
+# copy all the resources and python scripts to build directory
+mayaUsd_copyDirectory(${TARGET_NAME}
+    SOURCE ${CMAKE_CURRENT_SOURCE_DIR}
+    DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
+    EXCLUDE "*.txt"
+)
+
+foreach(script ${test_script_files})
+    mayaUsd_get_unittest_target(target ${script})
+    mayaUsd_add_test(${target}
+        PYTHON_MODULE ${target}
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+        ENV
+            "MAYA_DISABLE_CIP=1"
```

Comment:
I believe we can set below two Maya environment variables in `test.cmake`, so don't have to worry about setting them every time.

```
MAYA_DISABLE_CIP=1  Avoid fatal crash on start-up.
MAYA_DISABLE_CER=1  Customer Error Reporting.
```
These can be added to the list of ENV variables here:
```
    # without "MAYA_NO_STANDALONE_ATEXIT=1", standalone.uninitialize() will
    # set exitcode to 0
    set_property(TEST "${test_name}" APPEND PROPERTY ENVIRONMENT
        "MAYA_NO_STANDALONE_ATEXIT=1"
        "MAYA_DISABLE_CIP=1"
        "MAYA_DISABLE_CER=1")
```